src/framework/components/particlesystem/particlesystem_system.js

```diff
@@ -197,10 +197,35 @@ pc.extend(pc, function() {
             }, {
                 name: "spawnBounds",
                 displayName: "Spawn Bounds",
-                description: "Defines an AABB in which particles are allowed to spawn",
+                description: "Defines an AABB/radius in which particles are allowed to spawn",
```

Comment:
And ideally, we should try to get as much symmetry as possible between the collision component size properties and the particle system property names.